src/main/java/org/openpnp/machine/reference/ReferenceCamera.java

```diff
@@ -178,12 +181,37 @@ protected BufferedImage transformImage(BufferedImage image) {
         
         // apply affine transformations
         if (rotation != 0) {
-            // TODO: Fix cropping of rotated image:
-            // http://stackoverflow.com/questions/22041699/rotate-an-image-without-cropping-in-opencv-in-c
-            Point center = new Point(mat.width() / 2D, mat.height() / 2D);
-            Mat mapMatrix = Imgproc.getRotationMatrix2D(center, rotation, 1.0);
-            Imgproc.warpAffine(mat, mat, mapMatrix, mat.size(), Imgproc.INTER_LINEAR);
-	    mapMatrix.release();
+		double rot = rotation;
```

Comment:
@cri-s Can you put some comments in this block to explain what is going on? I find this code to be very confusing as there is a lot of manipulation of the rot variable and no explanation as to why. 

In addition, can you explain the reason for the limit of 13 degrees?